src/database/schema/ChatSchema.ts

```diff
@@ -20,7 +20,7 @@ class ChatSchema implements BaseDbSchema {
 
   updatedAt: string;
 
-  rawJson: string;
+  rawJson: any;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The type of `rawJson` has been changed from `string` to `any`. This could potentially introduce runtime errors if the code that uses `rawJson` expects it to be a string. If the change is necessary, ensure that all usage of `rawJson` is updated to handle the new type.
```diff
-  rawJson: string;
+  rawJson: any;
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->